LeaderboardBackend.Test/Runs.cs

```diff
@@ -65,21 +68,72 @@ public async Task SetUp()
             _categoryId = createdCategory.Id;
         }
 
+        [Test]
+        public static async Task GetRun_OK()
```

Comment:
I don't think test methods are meant to be static. Could be wrong though.

Reply:
I think dropping `static` really fixed my tests after all? Firstly this was a mistake, and secondly tests were constantly failing when comparing `CreatedAt`s, even though both values were equal.

Reply:
What the

I'll have to check when I'm back home. Currently away from my computer

Reply:
I didn't drop `static` from this test. I've removed it from all tests.